services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession.objects:
```

Comment:
I think it is better to `import` and use `QuerySet` instead of `MovieSession.objects`